src/Aspire.Hosting.Azure.AppContainers/ContainerAppExtensions.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Azure.Provisioning.AppContainers;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for customizing Azure Container App resource.
+/// </summary>
+public static class ContainerAppExtensions
```

Comment:
```suggestion
public static class AzureContainerAppExtensions
```

Reply:
This was intentional, since the type I am extending is `ContainerApp`. Thoughts?

Reply:
That's fine then.